### PR TITLE
Deprecation function and formatting

### PR DESCRIPTION
**Pull request by TheDiemer**
Added a function to enable deprecated wrappers to spew a warning when using a deprecated function (like query_cases) as well as formatting the tooling via [black code formatter](https://github.com/psf/black)

### CODE REVIEW

**Comment by TheDiemer** [comment]
@sferich888 MOST of the changes are re: the formatter, but there is the [deprecation function](https://github.com/TheDiemer/pyhydra/pull/12/files#diff-60cc9e4212aea4bfc879a155b5e2547fR140-R155) and [its application](https://github.com/TheDiemer/pyhydra/pull/12/files#diff-60cc9e4212aea4bfc879a155b5e2547fR447) that I'd love to have you give a once over if you can.

File: hydra/hydra.py [hunk 1]
```diff
@@ -49,355 +53,488 @@ def __get_api(self, endpoint, parameters=None, headers=None):
...
         try:
             response = r.json()
         except:
             response = r.text
-        if response == '': response = 'Deleting information from "{}" succeeded.'.format(endpoint)
+        if response == "":
+            response = 'Deleting information from "{}" succeeded.'.format(endpoint)
         return response
 
+    def deprecated(func):
```
Comment (sferich888) [inline hydra/hydra.py]: I'm not sure we want this as part of the class. 
Depending on how this grows having this as part of the class, may not make sense. 

@davemulford what are your thoughts here?

Comment (TheDiemer) [reply hydra/hydra.py]: That is fine with me. I am curious of Dave's thoughts, but it can be moved outside of the class easily.

Comment (TheDiemer) [reply hydra/hydra.py]: Per irc conversation with @davemulford, I moved from custom code for this over to [this prebuilt library](https://pypi.org/project/deprecation/).
This removes the need for the conversation wrt location of the function.

**Comment by TheDiemer** [comment]
@sferich888 can you review the changes? _Much_ cleaner with the library

File: hydra/hydra.py [hunk 2]
```diff
@@ -49,355 +53,475 @@ def __get_api(self, endpoint, parameters=None, headers=None):
...
-           query_params.update({'ownerSsousername': ", ".join(ownerSsousername)})
-       if tags:
-           query_params.update({'tags': ", ".join(tags)})
+        return self.__put_api("cases/{}/owner".format(case_number), payload=user)
+
+    @deprecation.deprecated(
+        deprecated_in="0.1",
+        removed_in="1.0",
+        details="Use the search_cases function instead",
+    )
```
Comment (sferich888) [inline hydra/hydra.py:434]: @davemulford you may have to help us learn to start versioning our pushes.

Comment (TheDiemer) [reply hydra/hydra.py:434]: That reminds me, I actually meant to update the version in the setup.py to 0.1

Comment (TheDiemer) [reply hydra/hydra.py:434]: @sferich888 is there something that you are looking for here before we merge?

Comment (sferich888) [reply hydra/hydra.py:434]: No, I may just need to be more deliberate about updates.

**Comment by sferich888** [comment]
> @sferich888 can you review the changes? _Much_ cleaner with the library

/lgtm

**Pull request merged**